marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/FlowController.java

```diff
@@ -118,6 +124,30 @@
         return ResponseEntity.ok(runStepsInAFlow(flowName, uploadedFiles, stepNumbers));
     }
 
+
+    @RequestMapping(value = "/{jobId}", method = RequestMethod.POST)
```

Comment:
Can you adjust the API to be something like "/stopJob/{jobId}"? Thanks

Reply:
thanks for the review, it was a good contribution